crates/tuono/src/typescript/types_jar.rs

```diff
@@ -0,0 +1,95 @@
+use crate::typescript::FileTypes;
+use glob::glob;
+use std::fs::read_to_string;
+use std::path::{Path, PathBuf};
+use tracing::error;
+
+const TUONO_MACRO_TRAIT_NAME: &str = "Type";
+
+#[derive(Debug, Clone, Default)]
```

Comment:
Do we need `Debug` for this?

```suggestion
#[derive(Clone, Default)]
```